packages/web-react/config/rollup.config.js

```diff
@@ -1,53 +1,60 @@
-import nodeResolve from '@rollup/plugin-node-resolve';
+// import nodeResolve from '@rollup/plugin-node-resolve';
```

Comment:
Let's maybe remove the line instead of commenting out?

Reply:
Removed.